integration-tests/rest-client-reactive-stork/pom.xml

```diff
@@ -121,6 +116,18 @@
             </resource>
         </resources>
         <plugins>
+            <plugin>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <annotationProcessorPaths>
+                        <path>
+                            <groupId>io.smallrye.stork</groupId>
+                            <artifactId>stork-configuration-generator</artifactId>
+                            <version>2.7.0</version>
```

Comment:
Could you try removing the version? With recent `maven-compiler-plugin` version, it should get it from the BOM.

Reply:
I tried before and I could swear I had failures without specifying the version here. Now it passes for some obscure reason

Reply:
We'll see what happens on CI.